python/GafferSceneUI/ShaderQueryUI.py

```diff
@@ -397,6 +450,15 @@ def __menuDefinition( self ) :
 
 		result = IECore.MenuDefinition()
 
+		result.append(
+			"/From Affected",
```

Comment:
Perhaps `From Location` makes more sense in this context?

Reply:
Fixed in 2f40e71128f8518f0ef9761b1b752872b836c7cd